data/json/items/tool/knives.json

```
@@ -92,5 +92,21 @@
     "color": "light_gray",
     "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 11 ] ],
     "flags": [ "STAB", "SHEATH_KNIFE" ]
+  },
+  {
+    "type": "GENERIC",
```

```suggestion
    "type": "GENERIC",
```
"type": "TOOL",

This item should probably be a tool?